internal/upload/awsupload/awsupload.go

```diff
@@ -243,6 +243,23 @@ func (a *AWS) Register(name, bucket, key string, shareWith []string, rpmArch str
 
 	log.Printf("[AWS] 🎉 AMI registered: %s", *registerOutput.ImageId)
 
+	// Tag the image with the image name.
+	req, _ := a.ec2.CreateTagsRequest(
```

Comment:
```suggestion
	req, _ = a.ec2.CreateTagsRequest(
```

Go doesn't like that you use `:=` without introducing a new variable.

Also, a quick test in `aws.sh` might be very cool. Otherwise, the change looks good! :-)

Reply:
Is this ignoring the error return?

Reply:
Ah, thank y'all for that. My golang is still improving.